bin/create-config

```diff
@@ -14,9 +15,33 @@ parser.add_argument('source', metavar='SOURCE-SHAPEFILE-PATH')
 parser.add_argument('boundary', metavar='NAME-OF_BOUNDARY')
 
 args = parser.parse_args()
+boundary_path = Path('./boundaries/build') / Path(args.boundary)
+source_path = Path(args.source)
 
+# Create a folder
 try:
-    boundary_path = Path('./boundaries/build') / Path(args.boundary)
     boundary_path.mkdir()
 except FileExistsError as e:
     sys.exit('{} already exists. Please pick a new name'.format(args.boundary))
+
+# Get existing json entries
+with open('./boundaries/boundary-config.json', 'r') as jsonFile:
+    try:
+        config_entries = json.load(jsonFile)
+    except json.decoder.JSONDecodeError as e:
+        config_entries = []
+
+# Make a new entry
+src_proj4 = proj4_from_shp(str(source_path))
+src_encoding = encoding_from_shp(str(source_path))
+new_config = BoundaryConfig(shp_path=str(source_path),
+                            proj4=src_proj4,
+                            encoding=src_encoding,
+                            boundary=args.boundary)
+
+# Append it to existing entries
+config_entries.append(new_config.dumps())
```

Comment:
This will create a strange file, where your JSON is an array where each element is serialized JSON. This will make it generally hard to understand. Instead, I'd change your `dumps` method on `BoundaryConfig` into an `as_dict` method which returns the `OrderedDict`, not serialized JSON.

Reply:
Another thing to think about here is if you want to detect that there's an existing entry for the boundary you're building? (Genuine question: I haven't thought about if you want to handle that specially or not.)